spaceship/static/wip-overlay.js

```diff
@@ -0,0 +1,45 @@
+(function() {
+  var closeOverlay = (evnt) => {
+    document.getElementById('overlay-background').style.display = 'none';
+    document.getElementById('overlay').style.display = 'none';
+    evnt.preventDefault();
+    evnt.stopPropagation();
+  };
+  // cookie functions based on https://stackoverflow.com/questions/14573223/set-cookie-and-get-cookie-with-javascript#24103596
+  var setCookie = (name, value, days) => {
```

Comment:
We have jquery in this project, maybe that has some better utilities to hand already?

Reply:
Ah, good to know. I will take a peak at the jQuery way.